english/strange-room/room.md

```diff
@@ -2,5 +2,5 @@ Upon entering, you see a sparsely furnished room with just a table, with a lit c
 
 It's obvious someone has been here recently but who?
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.